### PR TITLE
APIv4: GET /teams/{team_id}/members

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Implementation of endpoint APIv4: GET /teams/{team_id}/members

#### Ticket Link
GitHub issue: #5409 

#### Checklist
- [x] Added or updated unit tests (required for all new features)
- [x] Added API documentation (required for all new APIs)  [mattermost-api-reference-100](https://github.com/mattermost/mattermost-api-reference/pull/100)
- [x] All new/modified APIs include changes to the drivers

### CODE REVIEW

File: api4/team.go [hunk 1]
```diff
@@ -20,6 +20,7 @@ func InitTeam() {
 
 	BaseRoutes.Team.Handle("", ApiSessionRequired(getTeam)).Methods("GET")
 	BaseRoutes.Team.Handle("/stats", ApiHandler(getTeamStats)).Methods("GET")
+	BaseRoutes.Team.Handle("/members", ApiHandler(getTeamMembers)).Methods("GET")
```
Comment (jwilander) [inline api4/team.go]: Can we change `ApiHandler` to `ApiSessionRequired`?

Comment (saturninoabril) [inline api4/team.go]: Noted. I've updated route to session required, and test from forbidden to unauthorized.

**Comment by grundleborg** [comment]
@saturninoabril please could you rebase this onto current master to address the conflicts.

**Comment by saturninoabril** [comment]
@grundleborg done rebase. thanks!

**Pull request merged**